<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>459</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>400</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>PBCompressor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/images/icon.png</normaloff>:/images/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>Imagens</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QListWidget" name="listImagens">
         <property name="acceptDrops">
          <bool>true</bool>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Opções de Compressão</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Qualidade e Formato</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Qualidade:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="sliderQualidade">
                <property name="maximum">
                 <number>100</number>
                </property>
                <property name="value">
                 <number>80</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinQualidade">
                <property name="minimumSize">
                 <size>
                  <width>35</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::NoButtons</enum>
                </property>
                <property name="maximum">
                 <number>100</number>
                </property>
                <property name="value">
                 <number>80</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Formato de Saída:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboFormato">
                <item>
                 <property name="text">
                  <string>JPG</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>JPEG</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>PNG</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>BMP</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Redimensionamento</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QRadioButton" name="radioPorcentagem">
                <property name="text">
                 <string>Porcentagem</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Altura:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinAltPor">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="suffix">
                   <string>%</string>
                  </property>
                  <property name="maximum">
                   <number>100</number>
                  </property>
                  <property name="value">
                   <number>100</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Largura:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinLarPor">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="suffix">
                   <string>%</string>
                  </property>
                  <property name="maximum">
                   <number>100</number>
                  </property>
                  <property name="value">
                   <number>100</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QRadioButton" name="radioPixels">
                <property name="text">
                 <string>Tamanho Fixo (Pixels)</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Altura:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinAltPix">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="suffix">
                   <string> px</string>
                  </property>
                  <property name="maximum">
                   <number>10000</number>
                  </property>
                  <property name="value">
                   <number>3000</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Largura:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinLarPix">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="suffix">
                   <string> px</string>
                  </property>
                  <property name="maximum">
                   <number>10000</number>
                  </property>
                  <property name="value">
                   <number>2000</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>Opções de Saída</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Caminho:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineCaminho">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnCaminho">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>718</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionAbrir_Imagens"/>
    <addaction name="separator"/>
    <addaction name="actionAlternar_Tema"/>
    <addaction name="separator"/>
    <addaction name="actionSAir"/>
   </widget>
   <widget class="QMenu" name="menuAjuda">
    <property name="title">
     <string>Ajuda</string>
    </property>
    <addaction name="actionSobre_Qt"/>
    <addaction name="separator"/>
    <addaction name="actionSobre"/>
   </widget>
   <widget class="QMenu" name="menuCompress_o">
    <property name="title">
     <string>Compressão</string>
    </property>
    <addaction name="actionIniciar"/>
    <addaction name="separator"/>
    <addaction name="actionVoltar_aos_Padr_es"/>
    <addaction name="actionLimpar_Arquivos"/>
   </widget>
   <addaction name="menuArquivo"/>
   <addaction name="menuCompress_o"/>
   <addaction name="menuAjuda"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAbrir_Imagens"/>
   <addaction name="separator"/>
   <addaction name="actionIniciar"/>
   <addaction name="separator"/>
   <addaction name="actionLimpar_Arquivos"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbrir_Imagens">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
   </property>
   <property name="text">
    <string>Abrir Imagens..</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSAir">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
   <property name="toolTip">
    <string>Fechar o Programa</string>
   </property>
  </action>
  <action name="actionSobre">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/info.png</normaloff>:/images/info.png</iconset>
   </property>
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
  <action name="actionSobre_Qt">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/qt.png</normaloff>:/images/qt.png</iconset>
   </property>
   <property name="text">
    <string>Sobre Qt</string>
   </property>
  </action>
  <action name="actionIniciar">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/start.png</normaloff>:/images/start.png</iconset>
   </property>
   <property name="text">
    <string>Iniciar</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionLimpar_Arquivos">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/remove.png</normaloff>:/images/remove.png</iconset>
   </property>
   <property name="text">
    <string>Remover Arquivos Selecionados</string>
   </property>
  </action>
  <action name="actionVoltar_aos_Padr_es">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/back.png</normaloff>:/images/back.png</iconset>
   </property>
   <property name="text">
    <string>Voltar aos Padrões</string>
   </property>
  </action>
  <action name="actionAlternar_Tema">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/theme.png</normaloff>:/images/theme.png</iconset>
   </property>
   <property name="text">
    <string>Alternar Tema</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
